### PR TITLE
update witness class in springmvc-annotation-5.x-plugin

### PR DESCRIPTION
**Pull request by xzyJavaX**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #[11053](https://github.com/apache/skywalking/issues/11053)
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking-java/blob/main/CHANGES.md).

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Release Notes.
 * Implement new naming policies for names of auxiliary type, interceptor delegate field, renamed origin method, method
   access name, method cache value field. All names are under `sw$` name trait. They are predictable and unchanged after
   re-transform.
+* Update witness class in springmvc-annotation-5.x-plugin.
```
Comment (wu-sheng) [inline CHANGES.md:14]:
```suggestion
* Fix witness class in springmvc-annotation-5.x-plugin to avoid falling into v3 use cases.
```

**Review by wu-sheng — APPROVED** [review]

**Review by wu-sheng — CHANGES_REQUESTED** [review]
It looks like you are breaking Spring6, `Test / spring-6.x-scenario (pull_request)` is failing, please recheck.

**Comment by lujiajing1126** [comment]
I suppose the witness class for Spring 5.x should exist only in Spring MVC 5.x.

@wu-sheng Is there any toolkit to help find witness class?

**Review by wu-sheng — APPROVED** [review]
LGTM

**Pull request merged**